physical_property/logP/example_submission_file/logP-DanielleBergazinExampleFile-1.csv

```diff
@@ -9,32 +9,34 @@
 #
 # PREDICTION SECTION
 #
-# It is mandatory to submit  water to octanol (ΔG_octanol - ΔG_water) transfer free energy (TFE) predictions for all 22 molecules.
+# It is mandatory to submit water to octanol (ΔG_octanol - ΔG_water) transfer free energy (TFE) predictions for all 22 molecules.
 # Incomplete submissions will not be accepted.
 # The energy units must be in kcal/mol.
+
+# Please report the general molecule `ID tag` in the form of `SMXX` (e.g. SM25, SM26, etc).
+# Please indicate the microstate(s) used in the `Molecule ID/IDs considered (no commas)` section (e.g. `SM25_micro000`, SM25_extra001`)
```

Comment:
Your example here does not seem to be correctly formatted; there's an extra single quote or not enough single quotes.

Maybe also give TWO examples, one with only a single ID and one with two IDs.

Reply:
Good catch. I used `SM25_micro000` and `SM26_micro000 SM26_extra001` as new examples. There are also some examples in the "predictions" section.